R/pkratio-datamapping.R

```diff
@@ -10,35 +10,92 @@ PKRatioDataMapping <- R6::R6Class(
     sizeGrouping = NULL,
     shapeGrouping = NULL,
 
+    colorGroupingDataFrame = NULL,

```

Comment:
As discussed this morning. We can group those guys together to ensure a more cohesive API